src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -62,8 +58,11 @@
     -webkit-font-smoothing: subpixel-antialiased;
     -moz-osx-font-smoothing: auto;
 }
+.theme-dark .exception-summary ~ .sf-toolbar .sf-toolbarreset {
```

Comment:
using non-namespaced `theme-dark` class is a no-go in the toolbar styles. It would conflict with non-toolbar styles.

Reply:
I've removed this. We don't actually need it. Thanks.